library/scripts/equalize-and-expand.js

```diff
@@ -0,0 +1,57 @@
+jQuery(document).ready( function($) {
+
+	/*** EQUALIZE ELEMENT HEIGHTS ***/
+
+  $(window).resize(function(){
+
+    if (Modernizr.mq('(min-width: 1024px)')) {
+
+			// reset in case it's already been equalized
+			unEqualize('body.home section.hero > div');
+			unEqualize('body.home section.articles article a');
+			unEqualize('body.home section.block');
+
```

Comment:
Small nitpick, looks like the tabs/spaces are misaligned. Not a deal breaker.

Reply:
You're right. It was like that originally too – I just copied and pasted stuff, didn't edit at all. Probably won't fix at this time.